src/icons/box-tools/__tests__/IconBoxToolsInstall.test.tsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { shallow } from 'enzyme';
```

Comment:
Hm, i am guessing TypeScript doesn't like magical imports?

Reply:
Yeah, TypeScript throws the "cannot find name 'shallow'" error without this line